packages/services/api/src/modules/oidc-integrations/providers/oidc-integrations.provider.ts

```diff
@@ -145,6 +147,20 @@ export class OIDCIntegrationsProvider {
       } as const;
     }
 
+    const currentUser = await this.session.getViewer();
+    await this.auditLog.record({
+      eventType: 'ORGANIZATION_SETTINGS_UPDATED',
+      updatedFields: JSON.stringify({
+        createOIDCIntegrationForOrganization: true,
+      }),
```

Comment:
Here we can safe which properties were set for the initial version (and obviously hide/abbreviate secrets).